cezaraugusto/extension.js
# Add --zip flags to build command

When users run `npx extension build <path>`, add a few flags related to the distribution package:

* `--zip [boolean]`: whether or not to compress the extension into a ZIP file. Defaults to `false`
* `--zip-source [boolean]`: whether or not to include the source files in the ZIP file. Defaults to `false`
* `--zip-filename <string>`: specify the name of the ZIP file. Defaults to the extension name and version, like `my-ext-name-1.0.0.zip`.

<img width="921" alt="Screenshot 2024-06-06 at 19 57 12" src="https://github.com/cezaraugusto/extension.js/assets/4672033/7be13157-b895-491f-bec6-9b57cc70ddc5">

### How to test

1. Have an extension
2. Running `yarn extension build <extension-path> --zip` should generate a zip file from distribution files to `(projectPath)/dist/<browser>/<zip-name>.zip`
3. Running `yarn extension build <extension-path> --zip-source` should generate a zip file from source files to `(projectPath)/dist/<browser>/<zip-name>-source.zip`
4. Running `yarn extension build <extension-path> --zip --zip-filename=my-custom-name` should generate a zip file from distribution files (with custom name) to `(projectPath)/dist/<browser>/<custom-zip-name>.zip`.
